toolkit/tools/scheduler/schedulerutils/graphbuildstate.go

```diff
@@ -20,27 +20,29 @@ type nodeState struct {
 
 // GraphBuildState represents the build state of a graph.
 type GraphBuildState struct {
-	activeBuilds     map[int64]*BuildRequest
-	nodeToState      map[*pkggraph.PkgNode]*nodeState
-	failures         []*BuildResult
-	reservedFiles    map[string]bool
-	conflictingRPMs  map[string]bool
-	conflictingSRPMs map[string]bool
+	activeBuilds           map[int64]*BuildRequest
+	nodeToState            map[*pkggraph.PkgNode]*nodeState
+	failures               []*BuildResult
+	reservedFiles          map[string]bool
+	conflictingRPMs        map[string]bool
+	conflictingSRPMs       map[string]bool
+	supressConflictingRPMs bool
```

Comment:
I'm not sure, if this parameter belong here. It doesn't reflect the current state of the build and is actually pretty much immutable. I'd suggest keeping it inside the scheduler and only adding a new argument to the `RecordBuildResult` function.